### PR TITLE
MM-38698: Add limit to the CLI compliance export command

### PR DESCRIPTION
**Pull request by agnivade**
```release-note
A batchSize option has been added to the mattermost export
CLI command to limit the number of items exported. By default,
if it is not included, it exports all the posts.
```

https://mattermost.atlassian.net/browse/MM-38698

### CODE REVIEW

File: cmd/mattermost/commands/export.go [hunk 1]
```diff
@@ -71,9 +71,13 @@ func init() {
 	ScheduleExportCmd.Flags().Int("timeoutSeconds", -1, "The maximum number of seconds to wait for the job to complete before timing out.")
 
 	CsvExportCmd.Flags().Int64("exportFrom", -1, "The timestamp of the earliest post to export, expressed in seconds since the unix epoch.")
+	CsvExportCmd.Flags().Int("batchSize", -1, "The number of posts to export.")
```
Comment (isacikgoz) [inline cmd/mattermost/commands/export.go:74]: Batch size resonate something a bit different to me, would a name like `limit` would make more sense?

**Review by isacikgoz — APPROVED** [review]
Looks good, only a naming suggestion.

File: cmd/mattermost/commands/export.go (hunk 1 above)
Comment (agnivade) [reply cmd/mattermost/commands/export.go:74]: Yep

File: cmd/mattermost/commands/export.go [hunk 2]
```diff
@@ -71,9 +71,13 @@ func init() {
...
 
 	ActianceExportCmd.Flags().Int64("exportFrom", -1, "The timestamp of the earliest post to export, expressed in seconds since the unix epoch.")
+	ActianceExportCmd.Flags().Int("batchSize", -1, "The number of posts to export.")
```
Comment (noxer) [inline cmd/mattermost/commands/export.go:77]: We could add the description, that a negative number disables the limitation. Maybe useful for people trying use it in scripts. 2/5

**Review by noxer — APPROVED** [review]
LGTM

**Pull request merged**